mitmproxy/test/tflow.py

```diff
@@ -115,6 +115,26 @@ def tflow(client_conn=True, server_conn=True, req=True, resp=None, err=None):
     return f
 
 
+def tdummyflow(client_conn=True, server_conn=True, err=None):
+    class DummyFlow(flow.Flow):
```

Comment:
I thought about it, but IMO since this class is only used to create and return the dummyflow, I thought its cleaner to contain everything inside the function. But I don't have strong feelings about it - do you think I should move it out of the function?